src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.tsx

```diff
@@ -93,4 +92,6 @@ const WidgetFeedbackContentComponent = ({ widgetData, emptyText, classes }: Cont
     );
 };
 
-export const WidgetFeedbackContent: ComponentType<$Diff<ContentType, CssClasses>> = withStyles(styles)(WidgetFeedbackContentComponent);
+type PropsWithoutClasses = Omit<ContentType, 'classes'>;
+
+export const WidgetFeedbackContent = withStyles(styles)(WidgetFeedbackContentComponent) as ComponentType<PropsWithoutClasses>;
```

Comment:
use this patern: export const WidgetFeedbackContent =
    withStyles(styles)(WidgetFeedbackContentComponent) as ComponentType<ContentType>;